src/metrics/captures.ts

```diff
@@ -91,10 +98,23 @@ export class KDRatioMetric implements Metric {
       'Kill Death Ratios for each unambiguous piece: ' +
         JSON.stringify(this.kdRatios, null, 2)
     );
+    console.log('\n')
+    console.log("KDRs TAKING INTO ACCOUNT PIECE VALUES (Pawn 1 point, Knight 3 points, Bishop 3 points, Rook 5 points, Queen 9 points, King 1 point): ")
```

Comment:
should a king really be worth 1 point?

Reply:
good suggestion. per this wiki article, fighting value seems to be around 4pts in standard vals, so i'll change to 4 https://en.wikipedia.org/wiki/Chess_piece_relative_value